include/integratorxx/quadratures/radial/lmg.hpp

```diff
@@ -0,0 +1,127 @@
+#include <integratorxx/util/lambert_w.hpp>
+#include <integratorxx/quadratures/radial/radial_transform.hpp>
+
+namespace IntegratorXX {
+namespace lmg {
+
+// Eq 19 of LMG paper (DOI 10.1007/s002140100263)
+inline double r_upper_obj(int m, double alpha, double r) {
+  const double am_term = (m + 1.0) / 2.0;
+  const double g_term  = std::tgamma((m + 3.0) / 2.0);
+  const double x = alpha * r * r;
+  return g_term * std::pow(x, am_term) * std::exp(-x);
```

Comment:
`std::pow` is slow for a floating point exponent. Since we know the exponent is a half-integer, one can write this in terms of a square root and a call to integer `pow`.

Reply:
This is easy enough to add. will fix, but again, not performance critical, there needs to be a balance between readability and these types of optimizations - spending at few extra cycles on a `pow` call doesn't really matter here.

Reply:
On top of this, `r_upper_obj` is only used for debugging purposes (i.e. to check that the argument returns the requested precision to epsilon)

Reply:
Fixed